wikimedia/wikipedia-ios
# [YiR] Add personalized article total slide

**Phabricator:** https://phabricator.wikimedia.org/T376066

### Notes
This PR saves data for the personalized reading slide and creates the slide for viewing, if data is available to display. I am also setting the base slide localized strings, since I needed to do some similar handling for the personalized slide (which contains both personalized and collective count text). I also did some minor Core Data cleanup (first commit).

I am not 100% certain that what I did was the best approach on these strings, but my hope is that this will allow for _most_ of the base string to get translated, with some additional separate translated text (like "63.69 million articles") that is inserted wholesale. The separate text can then use whatever short-form large number translation makes the most sense to the translator. When we get final numbers, we'll need to do a lookup for any translations merged with the old number and manually ensure the latest number is used before release.

### Test Steps
1. Fresh install on simulator, ensure there's a France or Italy country code, and French or Italian is chosen as an app primary language.
2. Click through articles to ensure **six or more** articles appear in the History tab.
3. Go to Developer Settings and enable feature flag.
4. Background the app and foreground to trigger data processing. Tap Profile, Year in Review option should appear.
5. Tap Year in Review. You should see the new personalized reading count slide appear in place of the first collective slide.
